sdcm/argus_results.py

```diff
@@ -32,6 +32,126 @@
     }
 }
 
+DEFAULT_VALIDATION_RULES = {
+    "P90 write": ValidationRule(fixed_limit=5),
+    "P90 read": ValidationRule(fixed_limit=5),
+    "P99 write": ValidationRule(fixed_limit=10),
+    "P99 read": ValidationRule(fixed_limit=10),
+    # will fail sometimes: https://github.com/scylladb/scylla-cluster-tests/issues/9294
+    "Throughput write": ValidationRule(best_abs=2000),
+    "Throughput read": ValidationRule(best_abs=2000),
+}
+
+WORKLOAD_VALIDATION_RULES = {
+    "write": {
+        "default": DEFAULT_VALIDATION_RULES,
+        "_mgmt_repair_cli": DEFAULT_VALIDATION_RULES | {
+            "duration": ValidationRule(fixed_limit=3 * 3600)  # will fail for tablets (best 24811)
```

Comment:
we saw much much worse times in OSS, 6h and above in some

Reply:
true, these are based on enterprise. Moving to sct_config will allow us to define it per test.